src/formpack/reporting/export.py

```diff
@@ -643,6 +622,15 @@ def to_geojson(
             # been answered
             first_geo = True
             for geo_field in all_geo_fields:
+                # Handle the API query param of geo_question_name if present by
+                # passing all geo fields that don't match the specified
```

Comment:
maybe it's just me, but "passing" in programming makes me think of "passing along" more than "passing by", like passing arguments to a function. i'd say "skipping" in this case for clarity